hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmMetrics.java

```diff
@@ -324,8 +324,8 @@ public void testKeyOps() throws Exception {
       writeClient.commitKey(keyArgs, keySession.getId());
     } catch (Exception e) {
       //Expected Failure in preExecute due to not enough datanode
-      assertTrue(e.getMessage().contains("No enough datanodes to choose"),
-          e::getMessage);
+      assertThat(e.getMessage()).withFailMessage(e.getMessage())
+          .contains("No enough datanodes to choose");
```

Comment:
nit: custom fail message is not needed

```suggestion
      assertThat(e.getMessage()).contains("No enough datanodes to choose");
```

Reply:
fixed